rotkehlchen/api/v1/schemas.py

```diff
@@ -1670,32 +1670,51 @@ class ExternalServicesResourceDeleteSchema(Schema):
     services = fields.List(SerializableEnumField(enum_class=ExternalService), required=True)
 
 
-class ExchangesResourceEditSchema(Schema):
-    name = NonEmptyStringField(required=True)
+class _BinanceMarketsSchema(Schema):
+    """Common logic used when adding and editing exchange credentials"""
```

Comment:
This naming and docstring is off

```suggestion
class BinanceMarketsSchemaMixin(Schema):
    """Additional logic for adding/editing Binance exchanges credentials"""
```